Common/src/CConfig.cpp

```diff
@@ -1777,11 +1777,11 @@ void CConfig::SetConfig_Options() {
   /* DESCRIPTION: Activate The adaptive CFL number. */
   addBoolOption("CFL_ADAPT", CFL_Adapt, false);
   /* !\brief CFL_ADAPT_PARAM
-   * DESCRIPTION: Parameters of the adaptive CFL number (factor down, factor up, CFL limit (min and max), acceptable linear residual )
+   * DESCRIPTION: Parameters of the adaptive CFL number (factor down, factor up, CFL limit (min and max)[, acceptable linear residual][, starting iteration]).
    * Factor down generally <1.0, factor up generally > 1.0 to cause the CFL to increase when the under-relaxation parameter is 1.0
    * and to decrease when the under-relaxation parameter is less than 0.1. Factor is multiplicative. \ingroup Config*/
   default_cfl_adapt[0] = 1.0; default_cfl_adapt[1] = 1.0; default_cfl_adapt[2] = 10.0; default_cfl_adapt[3] = 100.0;
```

Comment:
```suggestion
  default_cfl_adapt[0] = 0.1; default_cfl_adapt[1] = 2.0; default_cfl_adapt[2] = 10.0; default_cfl_adapt[3] = 1e10;
```
I think it would be nice to have the default values be with the config_template(?), i.e. 0.1, 2.0, 10.0, 1e10, 0.001, 1 (github does not allow me to suggest that over the two lines)